src/NUnitTestAdapterTests/TestDiscoveryTests.cs

```diff
@@ -31,6 +31,8 @@ public void LoadMockassembly()
             Assert.That(NUnit.Tests.Assemblies.MockAssembly.Tests, Is.EqualTo(31),
                 "The reference to mock-assembly.dll appears to be the wrong version");
 
+            TestCases = new List<TestCase>();
```

Comment:
This isn't being used as far as I can see. I assume it is getting used in a subsequent PR?